src/test/resources/com/puppycrawl/tools/checkstyle/sariflogger/ExpectedSarifLoggerAbsoluteLinuxPath.sarif

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "downloadUri": "https://github.com/checkstyle/checkstyle/releases/",
+          "fullName": "Checkstyle",
+          "informationUri": "https://checkstyle.org/",
+          "language": "en",
+          "name": "Checkstyle",
+          "organization": "Checkstyle",
+          "rules": [
+          ],
+          "semanticVersion": "null",
+          "version": "null"
```

Comment:
Just pointing out this isn't a valid sarif file because of this `null`.

Reply:
True, though that was already the case in the existing files. From what I can see it would require changing all of the tests and a considerably more involved test setup.